src/graphics.c

```diff
@@ -172,3 +188,92 @@ void graphics_char(int x, int y, char ch, struct graphics_color fgcolor,
     return graphics_bitmap(x, y, FONT_WIDTH, FONT_HEIGHT, &fontdata[u],
                            fgcolor, bgcolor);
 }
+
+uint8_t graphics_verify_color_range(uint8_t x) {
```

Comment:
Yes, but ints below 0 and above 255 would get casted and we want behavior where they get bounded, not casted. This actually used to be in console. Seemed more appropriate in graphics.